include/wx/private/wxprintf.h

```diff
@@ -902,7 +902,7 @@ struct wxPrintfConvSpecParser
 
         // warn if we lost any arguments (the program probably will crash
         // anyhow because of stack corruption...)
-        if ( nargs == wxMAX_SVNPRINTF_ARGUMENTS )
+        if ( nargs >= wxMAX_SVNPRINTF_ARGUMENTS )
```

Comment:
I don't think it's really safer, it just hides a potential bug in the code as `nargs` is never supposed to become strictly greater than the limit, so it makes things less clear.

What exactly is Coverity's problem here?